src/arby.ts

```diff
@@ -155,15 +174,26 @@ if (!module.parent) {
     cleanup$: getCleanup$,
     initCEX$,
     verifyMarkets,
+    initDB$,
   }).subscribe({
     error: error => {
       if (error.message) {
         console.log(`Error: ${error.message}`);
       } else {
         console.log(error);
       }
-      process.exit(1);
+      closeDB$().subscribe({
```

Comment:
Whenever you need to subscribe within a subscribe callback you should probably be using a transformation operation instead (such as `mergeMap`).

Reply:
I changed this by adding a pipe to startArby observable and then calling closeDB observable within mergeMap, but I'm really unsure if it worked, because if I put console.log statements inside to mergeMap (that I call closeDB), they're never getting called, and never printing, can you please check @erkarl 

I followed here: https://rxjs-dev.firebaseapp.com/api/operators/mergeMap

Reply:
Line that I suspect is not working: https://github.com/ExchangeUnion/market-maker-bot/pull/125/files#diff-77a45f12e136f324d3c28fe24cdf3a531c7582103e96498a48462eaf7414a509R181